scripts/install/moviematch.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+if [[ ! -f /install/.plex.lock ]]; then
+    echo_warn "Plex server not installed"
+    echo_query "Please enter your Plex Server's address (e.g. domain.ltd:32400)"
+    read -r plexaddress
+else
+    plexaddress="localhost:32400"
+fi
+
+echo_info "Moviematch needs to connect to Plex via a Token. You can hear how to retrieve one here:\nhttps://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/"
+echo_query "Please enter your Plex Token"
+read -r plextoken
+
+#shellcheck source=sources/functions/deno
+. /etc/swizzin/sources/functions/deno
+install_deno
+
+mmatchDir="/opt/moviematch"
+
+echo_progress_start "Cloning moviematch"
+git clone https://github.com/LukeChannings/moviematch.git $mmatchDir >> $log
+echo_progress_done "Repo cloned"
+
+chmod +x $mmatchDir
+chmod o+rx -R $mmatchDir
+
+useradd moviematch --system -d "$mmatchDir" >> $log 2>&1
+sudo chown -R moviematch:moviematch $mmatchDir
+
+cat > $mmatchDir/.env << ENV
+PLEX_URL="http://$plexaddress"
+PLEX_TOKEN=$plextoken
+PORT=8420
+ENV
+
+echo_progress_start "Installing systemd service and starting moviematch"
+cat > /etc/systemd/system/moviematch.service << SYSTEMD
+[Unit]
+Description=Moviematch
+Documentation=https://swizzin.ltd/applications/moviematch
+After=network.target
+
+[Service]
+Type=simple
+User=moviematch
+WorkingDirectory=${mmatchDir}
+ExecStart=/usr/local/bin/deno run --allow-net --allow-read --allow-env ${mmatchDir}/src/index.ts
```

Comment:
The latest version also needs `--unstable` so that it can [handle SIGINT](https://deno.land/manual/examples/os_signals) correctly.

Reply:
Would this be sufficient?
```suggestion
ExecStart=/usr/local/bin/deno run --allow-net --allow-read --allow-env --unstable ${mmatchDir}/src/index.ts
```
I don't think it will necessarily cause issue on a seedbox if this application won't terminate "cleanly" so to speak. After all there is no database or any other writing to the disk. Is there something that this would add? I'll take your word for it :p

Thanks for having a look either way!

Reply:
Looks like it just doesn't run without it and restarts every 800ms lmao